docs/reference/content/bson/pojos.md

```diff
@@ -196,6 +196,79 @@ all serializable because they are bound to the concrete types `Integer`, `String
 On their own, instances of `GenericTree` or `GenericClass` are not serializable by the `PojoCodec`. This is because the runtime type parameter
 information is erased by the JVM, and the type parameters cannot be specialized accurately.
 
+The 3.6 release of the driver further improves generic support with the addition of PropertyCodecProviders. The `PropertyCodecProvider` API 
+allows type safe support of container types, by providing concrete type parameters for the generic types as declared in the POJO.
```

Comment:
Hyphen between "type" and "safe". 
Remove comma between "types" and "by"

Reply:
Done